### PR TITLE
[chore] Prepare release 0.112.0

### PR DESCRIPTION
**Pull request by opentelemetrybot**
The following commands were run to prepare this release:
- make chlog-update VERSION=v0.112.0
- sed -i.bak s/0[.]111[.]0/0.112.0/g versions.yaml
- make multimod-prerelease
- make multimod-sync

### CODE REVIEW

**Review by songy23 — APPROVED** [review]

**Comment by songy23** [comment]
`check-links` failed https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/11468084744/job/31912449256?pr=35941 though probably not caused by this PR

**Comment by evan-bradley** [comment]
I have a PR to fix that real quick here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35944

**Review by codeboten — APPROVED** [review]

**Comment by evan-bradley** [comment]
I can't merge main into this branch myself, but CI looks good so far (outside of what appears to be a flaky test failure), so I have a PR to merge in main: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35946

If that doesn't work, I'll recreate the PR. I think we'd be okay, but I would prefer to be cautious.

File: .github/CODEOWNERS
Comment (evan-bradley) [inline .github/CODEOWNERS:1]: This file shouldn't change, let me double-check what's going on.

**Comment by evan-bradley** [comment]
GitHub still thinks we're out-of-date. I'm not going to mess around with this, I'll just re-create the PR.

**Pull request closed without merging**